README.md

```diff
@@ -298,6 +305,25 @@ export TORCHSIM_TILE_M=512
 export TORCHSIM_TILE_N=512
 export TORCHSIM_TILE_K=512
 ```
+## Common Memory
+Common Memory (CMEM) is a new feature introduced in the latest TPUs (newer than TPUv3). Multiple cores share this memory, which provides high bandwidth. Reusable tensors are stored and loaded from CMEM to avoid off-chip traffic. CMEM functions similarly to an L2 Cache in GPUs.
+
+Our compiler, however, cannot yet determine which tensor is allocated in CMEM. To handle this limitation, we provide a CMEM allocation strategy API as a Python module.
```

Comment:
This line is good but how about this:
User can provide software manged allocation/eviction strategy for tensors